src/System.Windows.Forms/src/System/Windows/Forms/Controls/PropertyGrid/PropertyGridInternal/GridEntry.cs

```
@@ -1754,10 +1786,10 @@ void PaintOutlineWithClassicStyle(Graphics g, Rectangle r)
         PaintValueFlags paintFlags,
         string? text = null)
     {
-        PropertyGridView ownerGrid = OwnerGridView;
+        PropertyGridView ownerGrid = OwnerGridView!;
         Debug.Assert(ownerGrid is not null);
```

Let's convert this to throw `InvalidOperationException` if `OwnerGridView = null` to avoid getting an NRE